<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginUI</class>
 <widget class="QMainWindow" name="PluginUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_pCentralWidget">
   <layout class="QVBoxLayout" name="m_pLayout">
    <item>
     <widget class="QTabWidget" name="m_pTabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_pMenuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="m_pActionLoadPlugin"/>
    <addaction name="m_pActionAddTabs"/>
    <addaction name="m_pActionInitializePages"/>
    <addaction name="m_pActionExit"/>
   </widget>
   <addaction name="m_pMenuActions"/>
  </widget>
  <widget class="QStatusBar" name="m_pStatusBar"/>
  <action name="m_pActionLoadPlugin">
   <property name="text">
    <string>Load Plugin...</string>
   </property>
  </action>
  <action name="m_pActionAddTabs">
   <property name="text">
    <string>Add Tabs</string>
   </property>
  </action>
  <action name="m_pActionInitializePages">
   <property name="text">
    <string>Initialize Pages</string>
   </property>
  </action>
  <action name="m_pActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pActionExit</sender>
   <signal>triggered()</signal>
   <receiver>PluginUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
